templates/keepalived.conf.j2

```diff
@@ -50,8 +50,8 @@ vrrp_instance VI_1 {
 {% endfor %}
     }
     authentication {
-        auth_type PASS
-        auth_pass rke2servers
+        auth_type AH
```

Comment:
This should be configurable with `PASS` as the default. Also in case of `AH`, it would be good to have an additional option for `auth_algo`.

Reply:
I'm no expert here, I just thought the AH algo was better than the PASS algo because the AH algo isn't as sniffable.

Reply:
Because there are users which are already using this Ansible Role and by changing keepalived configuration the role may cause service interruption.